spec/integration/indexes_spec.rb

```diff
@@ -105,8 +105,7 @@ class Comment < ActiveRecord::Base; end
       end
 
       it 'executes the percona command' do
-        expect_percona_command('ADD INDEX `new_index_comments_on_some_id_field` (`some_id_field`)')
-        expect_percona_command('DROP INDEX `index_comments_on_some_id_field`')
+        expect_percona_command('RENAME INDEX `index_comments_on_some_id_field` TO `new_index_comments_on_some_id_field`')
```

Comment:
Should we update this to expect the command based on Rails version? (I assume you had to update this because Rails now send `RENAME INDEX` instead of `ADD INDEX`/`DROP INDEX`?)